plugins/carbon_black_defense/plugin.spec.yaml

```diff
@@ -109,357 +68,31 @@ types:
-      type: string
-    processId:
-      description: Process ID
-      title: Process ID
-      type: integer
-    targetCommandLine:
-      description: Target Command Line
-      title: Target Command Line
-      type: string
-    targetName:
-      description: Target Name
-      title: Target Name
-      type: string
-    targetPid:
-      description: Target Process ID
-      title: Target Process ID
-      type: integer
-    targetPrivatePid:
-      description: Target Private Process ID
-      title: Target Private PID
-      type: string
-    userName:
-      description: User Name
-      title: User Name
-      type: string
-  results:
-    alertCategory:
-      description: Alert Category
-      title: Alert Category
-      type: string
-    alertScore:
-      description: Alert Score
-      title: Alert Score
-      type: integer
-    attackStage:
-      description: Attack Stage
-      title: Attack Stage
-      type: string
-    createTime:
-      description: Create Time
-      title: Create Time
-      type: integer
-    deviceDetails:
-      description: Device Details
-      title: Device Details
-      type: deviceDetails
-    eventId:
-      description: Event ID
-      title: Event ID
-      type: string
-    eventTime:
-      description: Event Time
-      title: Event Time
-      type: integer
-    eventType:
-      description: Event Type
-      title: Event Time
-      type: string
-    incidentId:
-      description: Incident ID
-      title: Incident ID
+  policyAction:
+    action:
+      description: Policy Action
+      title: Policy Action
       type: string
-    longDescription:
-      description: Long Description
-      title: Long Description
-      type: string
-    netFlow:
-      description: Net Flow
-      title: Net Flow
-      type: netFlow
-    parentApp:
-      description: Parent Application
-      title: Parent Application
-      type: parentApp
-    processDetails:
-      description: Process Details
-      title: Process Details
-      type: processDetails
-    registryValue:
-      description: Registry Value
-      title: Registry Value
-      type: string
-    securityEventCode:
-      description: Security Event Code
-      title: Security Event Code
-      type: string
-    selectedApp:
-      description: Selected Application
-      title: Selected Application
-      type: parentApp
-    shortDescription:
-      description: Short Description
-      title: Short Description
-      type: string
-    targetApp:
-      description: Target Application
-      title: Target Application
-      type: parentApp
-    threatIndicators:
-      description: Threat Indicators
-      title: Threat Indicators
-      type: '[]string'
-  indicators:
     applicationName:
       description: Application Name
       title: Application Name
       type: string
-    indicatorName:
-      description: Indicator Name
-      title: Indicator Name
+    reputation:
+      description: Reputation
```

Comment:
This could benefit from a better description, if you happen to have one (I personally have no idea what Reputation means here)

Reply:
Haven't found a good description online, will reach out to CB

Reply:
Realized this is a subset of an unused type and removed it (along with the rest of the unused policyAction type)